LayoutTest/TestCase/LYTLayoutTestCase.m

```diff
@@ -67,15 +69,22 @@ - (void)runLayoutTestsWithViewProvider:(Class)viewProvider
                                                        self.viewUnderTest = view;
                                                        self.dataForViewUnderTest = data;
 
+                                                       numberOfCombinationsExecuted++;
+
                                                        // We must run this first to give the user a chance to add to viewsAllowingOverlap
                                                        validation(view, data, context);
+
+                                                       if (self.maxNumberOfCombinations != nil && numberOfCombinationsExecuted > self.maxNumberOfCombinations.integerValue) {
+                                                           [self failTest:@"Max number of combinations exceeded" view:view];
```

Comment:
Could you add maxNumberOfCombinations to the error message?

Reply:
That is a good idea